packages/openapi-to-graphql/src/schema_builder.ts

```diff
@@ -1026,80 +1026,63 @@ function linkOpRefToOpId<TSource, TContext, TArgs>({
 }
 
 /**
- * Determin if an argument should be created if the argument has already been
+ * Determine if an argument should be created if the argument has already been
  * provided through the options
  */
-function skipArg<TSource, TContext, TArgs>(
+function shouldSkipArgumentBecauseOptions<TSource, TContext, TArgs>(
```

Comment:
This is where the problem I mentioned before comes from.

`getResolver` in `resolver_builder.ts` uses `requestOptions` to change the API call.

`shouldSkipArgumentBecauseOptions` in `schema_builder.ts` uses `requestOptions` to determine if it should add an argument to the field in the GraphQL

They both provide different arguments.